src/pages/MyKiva/MyKivaPageContent.vue

```diff
@@ -8,6 +8,12 @@
 	/>
 	<MyKivaHero v-if="!userInHomepage" @show-navigation="handleShowNavigation" />
 	<MyKivaContainer>
+		<MyKivaProfile
+			class="tw-mt-4"
+			:lender="lender"
+			:user-info="userInfo"
+			v-if="!userInHomepage"
```

Comment:
I think we could remove this element as well, since we're only showing MyKiva on the homepage now.

Reply:
In that case, we can add a ticket to remove all cases we have for portfolio page, even removing `mykiva` component from that page.

Reply:
We can revisit when the experiment hopefully wins.